app/upgrades/v1rc7/testnet/upgrade.go

```diff
@@ -80,7 +80,7 @@ func MigrateFinalityProviders(goCtx context.Context, k btcstakingkeeper.Keeper)
 				SlashedBtcHeight:     fp.SlashedBtcHeight,
 				Jailed:               fp.Jailed,
 				HighestVotedHeight:   fp.HighestVotedHeight,
-				ConsumerId:           ctx.ChainID(), // Here we're not using fp.ConsumerId because the query result is not returning it. Also, this field is not available on release/v1.x branch. Make sure to remove it when backporting
+				ConsumerId:           fp.ConsumerId, // This field is not available on release/v1.x branch. Make sure to remove it when backporting
```

Comment:
if this is specific to phase-3, seems we don't need to backport this pr to `release/v1.x`